src/containers/ProjectsHomePage/ProjectsHomePage.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from '@/components';
+
+export const ProjectsHomePage = () => (
+  <section className="mx-auto mt-14 max-w-[1200px]">
+    <div className="mx-4 flex flex-col items-center justify-center lg:mx-14 xl:mx-4">
+      <h2 className="self-start pb-5 text-[28px] font-semibold text-neutral/90 md:text-[36px] ">Projekty</h2>
+      <div>carousel</div>
+
+      <Button kind="primary" text="Zobacz więcej" className="" />
+    </div>
+  </section>
+);
```

Comment:
What do you think about renaming those files and putting them in a `src/containers/HomePage/Projects.tsx` / `ProjectsSection.tsx`?

I'm asking because it wasn't so obvious to me if it's another page or something other :)